hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/RowCounter.java

```diff
@@ -65,22 +67,34 @@ public class RowCounter extends AbstractHBaseTool {
   private final static String OPT_END_TIME = "endtime";
   private final static String OPT_RANGE = "range";
   private final static String OPT_EXPECTED_COUNT = "expectedCount";
+  private final static String OPT_COUNT_DELETE_MARKERS = "countDeleteMarkers";
 
   private String tableName;
   private List<MultiRowRangeFilter.RowRange> rowRangeList;
   private long startTime;
   private long endTime;
   private long expectedCount;
+  private boolean countDeleteMarkers;
   private List<String> columns = new ArrayList<>();
 
   /**
    * Mapper that runs the count.
    */
   static class RowCounterMapper extends TableMapper<ImmutableBytesWritable, Result> {
 
-    /** Counter enumeration to count the actual rows. */
+    /** Counter enumeration to count the actual rows, cells and delete markers. */
     public static enum Counters {
```

Comment:
This class is interface public and  enum Counters is also public. i am not sure if it is fine to change this. may be extend?

Reply:
Maybe I am wrong this is fine. I see similar change in https://github.com/apache/hbase/commit/cda5a87d854e306c3d157c6d793d53451d6b168d